controller/tests/controller_server/utils_test.py

```diff
@@ -248,19 +245,13 @@ def test_validate_publish_volume_request(self, validate_capabilities, validate_s
             self.assertTrue("msg1" in ex.message)
 
         validate_capabilities.side_effect = None
+        validate_secrets.side_effect = [ValidationException("secrets")]
```

Comment:
could you rename it to `validate_secret` (singular) while you're at it?

Reply:
actually the original function should probably renamed to `validate_secrets` (plural) as its mock name.
what do you think?